src/psyclone/generator.py

```diff
@@ -542,7 +555,8 @@ def main(arguments):
     else:
         logging.basicConfig(level=loglevel)
     logger = logging.getLogger(__name__)
-    logger.debug("Logging system initialised.")
+    logger.debug("Logging system initialised. Default level is %s.",
+                 args.log_level)
```

Comment:
Do you mean: "Logging level is %s" (it is not always the default, sometimes it is the given by the argument)

Reply:
Oops. Yes, I did mean that. Fixed.